_topic_maps/_topic_map.yml

```diff
@@ -451,6 +451,8 @@ Topics:
     File: installing-openstack-installer-restricted
   - Name: OpenStack Cloud Controller Manager reference guide
     File: installing-openstack-cloud-config-reference
+  - Name: Deploying OpenStack with rootVolume and etcd on local disk
```

Comment:
we're not really deploying OpenStack here, but rather deploying an OpenShift cluster on OpenStack, with rootVolume, etc.